### PR TITLE
doc-changes

### PR DESCRIPTION
**Pull request by aaryaattrey**
1. Change on_event -> OnEvent in typed service.
2. Support config file loading from standard input.
3. Add default_max_records_num parameter in api config.

### CODE REVIEW

File: docs/configuration/other.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ api:
     ...
   api_security: !Jwt
       jwt_token 
+  default_max_records_num:
```
Comment (chubei) [inline docs/configuration/other.md:18]:
```suggestion
  default_max_records_num: 50
```
Change this so people know it's an ingeter

File: docs/configuration/other.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Other Settings
 
 ## API Configuration
-The `api` section of the Dozer configuration specifies how different types of APIs are set up and interact with other services. This section is organized into three main blocks, each corresponding to a specific type of API: `rest`, `grpc`, and `app_grpc`. Additionally, there's a `api_security` block which designates the security type to be applied across these API interfaces.
+The `api` section of the Dozer configuration specifies how different types of APIs are set up and interact with other services. This section is organized into three main blocks, each corresponding to a specific type of API: `rest`, `grpc`, and `app_grpc`. Additionally, there's a `api_security` block which designates the security type to be applied across these API interfaces. You can also configure the maximum number of records returned from queries using the `default_max_records_num` parameter.
```
Comment (chubei) [inline docs/configuration/other.md:4]: "maximum" -> "default limit of"

File: docs/configuration/other.md [hunk 3]
```diff
@@ -84,3 +85,7 @@ The `app_grpc` section configures the App gRPC API server. This server is used f
 | `cors`         | Boolean | Dictates if CORS is active for the app-specific gRPC API server. Default: `true`.                                                       |
 | `web`          | Boolean | Activates HTTP/1 + web support for the app-specific gRPC API server. Default: `true`.                                                   |
 | `enabled`      | Boolean | Indicator to enable or disable the app-specific gRPC API server. Default: `true`.                                                       |
+
+### Default Query Limit
+
+The `default_max_records_num` is an   `Integer` parameter that specifies the maximum number of records returned from queries. The default value is 50.
```
Comment (chubei) [inline docs/configuration/other.md:91]: "maximum" -> "default limit of"

Too many spaces before `Ingeter`?

**Review by chubei — CHANGES_REQUESTED** [review]

File: docs/configuration/other.md (hunk 1 above)
Comment (aaryaattrey) [reply docs/configuration/other.md]: addressed

File: docs/configuration/other.md (hunk 2 above)
Comment (aaryaattrey) [reply docs/configuration/other.md]: addressed

File: docs/configuration/other.md (hunk 3 above)
Comment (aaryaattrey) [reply docs/configuration/other.md]: addressed.

**Review by chubei — APPROVED** [review]

**Pull request merged**